### PR TITLE
JBEAP-18074 - OpenJ9 EAP build

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/JBEAP-18074
Signed-off-by: Ken Wills <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -77,15 +78,15 @@ modules:
...
+          - name: jboss.container.util.logging.bash
           - name: jboss.container.eap.prometheus.config
             version: '7.2'
           - name: os-eap-txnrecovery.run
             version: 'python3'
           - name: os-eap-python
             version: '3.6'
-packages:
-      content_sets_file: content_sets.yml
+            #packages:
```
Comment (jmesnil) [inline image.yaml]: is this comment on purpose?

**Review by jmesnil — COMMENTED** [review]
LGTM except my question about the comment for the content_sets.yml

File: image.yaml (hunk 1 above)
Comment (luck3y) [reply image.yaml]: Nope, thanks for catching, will update in a sec.

**Comment by luck3y** [comment]
@jmesnil Corrected, and also I updated the relevent override files to only build for their specified arches so we can avoid a surpise later ;)

**Review by jmesnil — DISMISSED** [review]
LGTM

**Comment by luck3y** [comment]
@jmesnil Sorry, I added the templates / imagestream updates here too, can you have another look?

File: templates/eap72-openjdk11-image-stream.json [hunk 2]
```diff
@@ -111,6 +111,69 @@
...
                 ]
             }
+        },
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "jboss-eap72-openj9-11-openshift-rhel8",
+                "annotations": {
+                    "openshift.io/display-name": "Red Hat JBoss EAP 7.2 OpenJ9 11 UBI 8 Image",
```
Comment (jmesnil) [inline templates/eap72-openjdk11-image-stream.json:121]: Should it be "UBI 8 Image" or "RHEL 8 Image"?

Comment (luck3y) [reply templates/eap72-openjdk11-image-stream.json:121]: The RCM default is to just use -rhel7 / -rhel8 so there aren't too many variants of the names.

Comment (luck3y) [reply templates/eap72-openjdk11-image-stream.json:121]: Ping for approval so I can merge @jmesnil :)

**Review by jmesnil — DISMISSED** [review]

**Comment by luck3y** [comment]
@jmesnil This is all set for handing over for building now I think, can you take a final look?

FYI @rgdoliveira @dbkreling

**Review by jmesnil — APPROVED** [review]

**Comment by mackaypeter** [comment]
@luck3y what's the reason behind the *IMAGE_NAME template changes here? If we're gonna introduce a new template parameter, shouldn't we do it for the JDK8 stream as well? And also for 7.3?

**Pull request merged**